### PR TITLE
bpo-34711: Fix test_httpservers for AIX (trailingSlashOK)

### PR DESCRIPTION
**Pull request by aixtools**
https://bugs.python.org/issue34711

### CODE REVIEW

File: Lib/test/test_httpservers.py [hunk 1]
```diff
@@ -29,6 +30,10 @@
 import unittest
 from test import support
 
+# Some OS accept trailingSlash as part of a filename
+# https://bugs.python.org/issue34711#msg325532 mentions Windows sometimes accepts a trailingSlash
+# so more work could be done
+trailingSlashOK = platform.system() in ('AIX')
```
Comment (nascheme) [inline Lib/test/test_httpservers.py:36]: Probably should use a literal set here.  If you meant to use a tuple, you forgot the trailing comma.  So, I suggest `in {'AIX'}` instead.

Comment (merwok) [inline Lib/test/test_httpservers.py:36]: This is a string, not a one-element tuple.

Equality test may be clearer (and simple to change if needed).

**Comment by aixtools** [comment]
Closing. Per discussion in python-bugs (https://bugs.python.org/issue34711) shall address this issue in Lib/http/server.py

**Comment by aixtools** [comment]
On 03/10/2018 02:16, Neil Schemenauer wrote:
> nascheme commented on this pull request.
>
>
>
>> @@ -29,6 +30,10 @@
>  import unittest
>  from test import support
>  
> +# Some OS accept trailingSlash as part of a filename
> +# https://bugs.python.org/issue34711#msg325532 mentions Windows sometimes accepts a trailingSlash
> +# so more work could be done
> +trailingSlashOK = platform.system() in ('AIX')
>
> Probably should use a literal set here.  If you meant to use a tuple, you forgot the trailing comma.  So, I suggest `in {'AIX'}` instead.
>
Yes, that is what I intended - as a hint that other platforms could be
added. E.g., maybe Windows could also just be added (so it becomes
{'AIX', 'Windows'} because 'sometimes' Windows accepts a trailing slash.

What is guideline when there is a good reason to think it may be valid
for a platform, but cannot test it personally?

Note: this resolution path (fix in the test) has been dropped. Still
thanks for alerting me to the proper brackets!

**Pull request closed without merging**